foundation-extensions/experience-platform-extension/edge-network-api-reference.md

```diff
@@ -178,20 +193,22 @@ MobileCore.registerExtensions([Edge.self, ...], {
 
 Use the AEPMobileCore API to register the Edge extension.
 
-#### Syntax
+**Syntax**
 
-```text
-// Swift MobileCore
-public static func registerExtensions(_ extensions: [Extension.Type], _ completion: (() -> Void)? = nil)
+```objectivec
++ (void) registerExtensions: (NSArray<Class*>* _Nonnull) extensions 
+                  completion: (void (^ _Nullable)(void)) completion;
 ```
 
-#### Example
+**Example**
+
+```objectivec
+@import AEPEdge;
+
+[AEPMobileCore registerExtensions:@[AEPMobileEdge.class] completion:nil];...
 
-```text
-[AEPMobileCore registerExtensions:@[AEPMobileEdge.class, ...] completion:^{
-  // processing after registration
-}];
 ```
+
 {% endtab %}
 {% endtabs %}
```

Comment:
@kevinlind should we add resetIdentities to this extension too?

Reply:
I added resetIdentities API
For now, I have the description 
"Clears all the states stored in the Edge extension."